server/embed/config.go

```diff
@@ -356,10 +362,14 @@ type Config struct {
 	// AuthTokenTTL in seconds of the simple token
 	AuthTokenTTL uint `json:"auth-token-ttl"`
 
-	ExperimentalInitialCorruptCheck     bool          `json:"experimental-initial-corrupt-check"`
-	ExperimentalCorruptCheckTime        time.Duration `json:"experimental-corrupt-check-time"`
-	ExperimentalCompactHashCheckEnabled bool          `json:"experimental-compact-hash-check-enabled"`
-	ExperimentalCompactHashCheckTime    time.Duration `json:"experimental-compact-hash-check-time"`
+	// Deprecated in v3.6, and will be decommissioned in v3.7.
+	ExperimentalInitialCorruptCheck bool          `json:"experimental-initial-corrupt-check"`
```

Comment:
This change was missed when migrating the `--experimental-initial-corrupt-check`, so please do it separately.  So that we can use this PR as an complete & independent & good example for other contributors reference on how to migrate experimental flags to server level feature gate.

Reply:
sg